tests/test-cases/cli/inspect/itis-industry.csv-metadata.json

```diff
@@ -71,7 +71,7 @@
     },
     "rdfs:seeAlso": [
         {
-            "@id": "http://gss-data.org.uk/data/gss_data/trade/ons-international-trade-in-services#scheme/itis-industry/dataset",
+            "@id": "http://gss-data.org.uk/data/gss_data/trade/ons-international-trade-in-services#scheme/itis-industry",
```

Comment:
Why was this change needed?

Reply:
IIRC, this was because this code list is a very old one where the `dcat:Dataset` and `skos:ConceptScheme` were seperate resources. This is not how csvcubed generates outputs at the moment, so we've altered the input to the style we're expecting.